Dockerfile

```diff
@@ -3,9 +3,9 @@ RUN mkdir -p /go/src/github.com/asecurityteam/asset-inventory-api
 WORKDIR $GOPATH/src/github.com/asecurityteam/asset-inventory-api
 COPY --chown=sdcli:sdcli . .
 RUN sdcli go dep
-RUN go get -u github.com/gobuffalo/packr/v2/packr2
+RUN GO111MODULE=on go get github.com/gobuffalo/packr/v2/packr2
```

Comment:
Task: Nice find! I believe you'll also need to add `GOFLAGS='-mod=vendor'` so go uses the vendor directory created by the `sdcli go dep` stage. See: https://github.com/golang/go/wiki/Modules#how-do-i-use-vendoring-with-modules-is-vendoring-going-away

Reply:
Ok. Added the parameter. Thank you for your opinion.
 @zlozano